_rules/lists-follow-content-model-a73be2.md

```diff
@@ -0,0 +1,248 @@
+---
+id: a73be2
+name: List elements follow content model
+rule_type: atomic
+description: |
+  This rule checks that list elements follow content model
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility tree
+  - CSS Styling
+  - DOM Tree
+acknowledgments:
+  authors:
+		- Brian Bors
+		- Wilco Fiers
+  previous_authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+The rule applies to every `ul`, `ol`, `dl`, and `menu` [HTML elements][] that is [included in the accessibility tree][] and has the same [semantic role][] as its [implicit semantic role][].
+
+## Expectation 1
+
+Each test target only has [child][] [node][] in the [flat tree][] for which one of the following is true:
+
+- the [child][] [element][] follows the [content model][]; or
+- the [child][] has an [explicit role][] which is the [Implicit ARIA semantics]() for an HTML element included in the [content model][] for the test target; or
```

Comment:
`dl` accept `div` in their content model, and `div` have no role. I'm not sure how this articulate with the expectation.